src/ocr-engine.ts

```diff
@@ -120,6 +120,39 @@ export class OCREngine {
     this._engine = null;
   }
 
+  /**
+   * Get the value, represented as a string, of a Tesseract configuration variable.
+   *
+   * For a list of configuration variables, see
```

Comment:
I'd suggest to just point to `setVariable` for the documentation, rather than duplicating this information. eg. `See {@link setVariable} for information on available variables.